lib/services/internalEngine/index.js

```diff
@@ -44,29 +44,35 @@
 'use strict';
```

Comment:
:warning: Do not review this file, it will be replaced by the elasticsearch service in another PR